nordicsemi/__main__.py

```diff
@@ -350,15 +350,15 @@ def display(key_file, key, format, out_file):
             kfile.write(kstr)
 
 
-@cli.group(short_help='Generate a Device Firmware Update package.')
+@cli.group(short_help='Display or generate a DFU package (zip file).')
```

Comment:
I dont see any updates for displaying a package.
Is it a good description?

Reply:
Yes, it's good. This is the docstring for the `nrfutil pkg` command.

The `nrfutil pkg` command has two subcommands: `nrfutil pkg display` and `nrfutil pkg generate`, so this is actually a correction.

Reply:
👍